### DISCUSSION TITLE
hamburger not working

### DISCUSSION THREAD

**Discussion by cedricbompart** [mmistakes/minimal-mistakes#2854] [Q&A]
- Minimal Mistakes version:
- Ruby gem or remote theme version:
> I'm using -> remote_theme: mmistakes/minimal-mistakes
- Jekyll version:
- Git repository URL:
> https://github.com/marcbompart/marcbompart.github.io
- GitHub Pages hosted (if yes provide URL to site):
> www.cabinetbompart.fr
- Operating system:

## Expected behavior
I've just recently updated my GitHub hosted pages. The hamburger is not showing anymore and the navigation links are squashed together in portrait mode.
I'm using the latest theme without any specific tags. The previous update was around September last year so I'm not sure which version was used during the last deployment (maybe 4.20.2).
I've overridden some of your layouts but I've checked this morning and I can't see any major changes which could prevent the hamburger to display properly.

**Comment by mmistakes** [comment]
The priority plus (greedy nav) isn't working on your site for some reason. A quick look at your site doesn't show any JavaScript errors in the console, and it does appear to be using the latest version of the theme's files/assets.

From what I see that you've modified in your repo, none of the layouts altered the navigation HTML or the JavaScript used. Not really sure what the issue is.

My suggestion would be to temporarily remove all your locally modified theme files (`_layouts`, `_includes`, `assets/css`) and build your site locally. See if the menu collapses on smaller viewports. If it works as expected, then slower start adding in your modified files to determine which one is out of sync with the theme.
